README.md

```diff
@@ -164,17 +164,32 @@ tofu destroy
 
 ## Further Documentation
 
-- [Terraform OCI Provider](https://www.terraform.io/docs/providers/oci/index.html)
+### Terraform Provider
 - [Oracle Cloud Infrastructure Provider](https://registry.terraform.io/providers/oracle/oci/latest/docs)
 - [Azure Active Directory Provider](https://registry.terraform.io/providers/hashicorp/azuread/latest/docs)
 - [Azure Provider](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs)
+- [AzAPI Provider](https://registry.terraform.io/providers/Azure/azapi/latest/docs)
+
+### Terraform Modules
+- [Azure Verified Modules](https://azure.github.io/Azure-Verified-Modules/indexes/terraform/)
+- [OCI Landing Zones](https://github.com/oci-landing-zones/)
 
 **Acknowledgement:** Code derived adapted from samples, examples and documentations provided by above mentioned providers.
 
+## Help
+
+Open an issue in this repository.
+
 ## Contributing
 
-Learn how to [contribute](./CONTRIBUTING.md).
+This project welcomes contributions from the community. Before submitting a pull request, please [review our contribution guide](./CONTRIBUTING.md).
+
+## Security
+
+Please consult the [security guide](./SECURITY.md) for our responsible security vulnerability disclosure process.
 
 ## License
 
-Copyright (c) 2017, 2024 Oracle Corporation and/or its affiliates. Licensed under the [Universal Permissive License 1.0](./LICENSE) as shown at [https://oss.oracle.com/licenses/upl](https://oss.oracle.com/licenses/upl/).
\ No newline at end of file
+Copyright (c) 2023,2024 Oracle and/or its affiliates.
```

Comment:
why change the dates? Why have two dates? See `(c)` comment above, it's invalid to include this as well as the word Copyright.

Reply:
It's [required/standard phrase](https://github.com/oracle/template-repo/blob/main/README.md) from OGHO

Reply:
but, again, legally incorrect.

Reply:
I'm no expert on legal and can't make any comment. But both license.txt and (c) are commonly used (if not all) in our repos. here an [example](https://github.com/oracle/heatwave)